Azure.ResourceManager.Core/SubscriptionOperations.cs

```diff
@@ -36,22 +36,22 @@ internal SubscriptionOperations(AzureResourceManagerClientOptions options, Resou
         internal ResourceGroupsOperations RgOperations => GetClient<ResourcesManagementClient>((uri, cred) =>
             new ResourcesManagementClient(uri, Id.Subscription, cred, ClientOptions.Convert<ResourcesManagementClientOptions>())).ResourceGroups;
 
-        public ResourceGroupOperations ResourceGroup(ResourceGroupData resourceGroup)
+        public ResourceGroupOperations GetResourceGroupOperations(ResourceGroupData resourceGroup)
```

Comment:
return ResourceGroup
```suggestion
        public ResourceGroup GetResourceGroup(ResourceGroupData resourceGroup)
```